core/src/test/scala/org/apache/spark/sql/delta/OptimisticTransactionSuite.scala

```diff
@@ -754,4 +754,18 @@ class OptimisticTransactionSuite extends QueryTest with SharedSparkSession {
       }
     }
   }
+
+  test("isolation level shouldn't be null") {
+    withTempDir { tempDir =>
+      val log = DeltaLog(spark, new Path(tempDir.getCanonicalPath))
+      // Initialize the log.
+      log.startTransaction().commitManually()
+
+      val txn = log.startTransaction()
+      txn.commit(addA :: Nil, ManualUpdate)
+
+      val isolationLevels = log.history.getHistory(Some(10)).map(_.isolationLevel)
+      assert(isolationLevels === Seq(Option("Serializable"), Option("SnapshotIsolation")))
```

Comment:
Why are they `Option`s since they're `Some` actually. I'd also use their object names:

```
import org.apache.spark.sql.delta._
Seq(Serializable, SnapshotIsolation).map(_.toString).map(Some.apply)
```

Reply:
good catch!